dv/uvm/core_ibex/sim.py

```diff
@@ -190,6 +193,74 @@ def get_simulator_cmd(simulator, yaml_path, enables):
             subst_cmd(entry['sim']['cmd'], enables, entry['sim'], env_vars))
 
 
+def filter_tests_by_config(cfg, test_list):
+    '''Filter the provided testlist to ensure all required RTL parameters are
+       provided by the Ibex core configuration.
```

Comment:
Nit: The usual docstring style is to start with a single-line sentence (a bit like a git commit message)

Reply:
fixed, thanks!